src/utils/daos.c

```diff
@@ -719,6 +733,16 @@ common_op_parse_hdlr(int argc, char *argv[], struct cmd_args_s *ap)
 			if (ap->dst == NULL)
 				D_GOTO(out_free, rc = RC_NO_HELP);
 			break;
+		case 'I':
+			D_STRNDUP(ap->dfs_prefix, optarg, strlen(optarg));
+			if (ap->dfs_prefix == NULL)
+				D_GOTO(out_free, rc = RC_NO_HELP);
+			break;
+		case 'H':
+			D_STRNDUP(ap->dfs_path, optarg, strlen(optarg));
+			if (ap->dfs_path == NULL)
+				D_GOTO(out_free, rc = RC_NO_HELP);
+			break;
```

Comment:
We should probably add corresponding FREE's to the args_free function in this file, which will free these arguments in case of any error

Reply:
sure